packages/@react-aria/searchfield/docs/useSearchAutocomplete.mdx

```diff
@@ -466,12 +467,11 @@ function ControlledSearchAutocomplete() {
 
   // Specify how each of the SearchAutocomplete values should change when an
   // option is selected from the list box
-  let onSelectionChange = (key) => {
+  let onSubmit = (value, key) => {
```

Comment:
hitting enter on the controlled search autocomplete after making a selection clears the field, not sure why

Reply:
Added to follow-up (#2286)